packages/x-date-pickers/src/MonthCalendar/PickersMonth.tsx

```diff
@@ -29,17 +26,16 @@ interface PickersMonthProps extends ExportedPickersMonthProps {
   selected?: boolean;
   value: number;
   tabIndex: number;
+  monthsPerRow: 3 | 4;
 }
 
-interface PickersMonthOwnerState extends PickersMonthProps {
-  wrapperVariant: WrapperVariant;
-}
+type PickersMonthOwnerState = PickersMonthProps;
 
 const useUtilityClasses = (ownerState: PickersMonthOwnerState) => {
```

Comment:
What do you think about dropping `ownerState` and just using `props` here? 🤔

Reply:
I'm fine with it
The core uses `const ownerState = props` fairly often (and never `ownerState={props}`)
But as long as we try to be coherent inside X.